UPGRADING.md

```diff
@@ -342,6 +342,28 @@ And here is the full difference between the report and event interface:
   }
 ```
 
+### Reset event count
+
+The method to reset the event count, preventing the `maxEvents` limit from being hit has been renamed:
+
+```diff
+- bugsnagClient.refresh()
++ Bugsnag.resetEventCount()
+```
+
+### Session endpoint
+
+Previously it was valid to supply a `notify` endpoint without supplying a `sessions` endpoint. Now if you supply one, you must supply the other.
```

Comment:
Is it worth re-iterating that this is intended for on-premise use, rather than concern all users?